packages/features/insights/server/routing-events.ts

```diff
@@ -408,6 +409,8 @@ class RoutingEventsInsights {
       }, {} as Record<string, string | undefined>);
 
       return {
+        "Booking UID": item.bookingUid,
+        "Booking Link": item.bookingUid ? `${WEBAPP_URL}/booking/${item.bookingUid}` : "",
```

Comment:
Does it make sense to do something like this? @eunjae-lee 

```suggestion
        ...( item.bookingUid & { "Booking Link": `${WEBAPP_URL}/booking/${item.bookingUid}` }), 
```

Reply:
hey @hbjORbj this part prepares data for CSV download, so we need "Booking Link" property always there for the csv result :)

Reply:
Gottcha!